it/EnterpriseAcceptanceSpec.scala

```diff
@@ -51,8 +61,14 @@ class EnterpriseAcceptanceSpec extends ServerAcceptanceSpec with WithWireMockHBa
       response.status shouldBe OK
       response.header("Content-Type") shouldBe Some(JSON)
       response.json.as[Enterprise] shouldBe
-        Enterprise(ern = TargetErn, entref = SampleEnterpriseReference, name = SampleEnterpriseName, postcode = SamplePostcode,
-          legalStatus = SampleLegalStatus, employees = Some(SampleNumberOfEmployees), jobs = Some(SampleJobs))
+        Enterprise(ern = TargetErn, entref = Some(SampleEnterpriseReference), name = SampleEnterpriseName,
+          tradingStyle = Some(SampleTradingStyle), address = aAddressSample(
+            line2 = Some(SampleAddressLine2),
```

Comment:
How does the address not have a line1 (which looks to be in the sample json), and is mandatory?